### PR TITLE
Added new lock events. Added analytics integrations

### PR DESCRIPTION
**Pull request by ntotten**
- Adds new events for lock 10
- Added docs on facebook & google analytics integrations

### CODE REVIEW

File: articles/analytics/integrations/_usage.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+## Usage
+After installation on your site there is nothing else you need to do to start collecting data. Auth0 Analytics will immedately begin sending events to <%- name %>.
+
+You will see the following events being logged:
+
+* Auth0 Lock show
```
Comment (mpaktiti) [inline articles/analytics/integrations/_usage.md:6]: @ntotten Do we have to repeat the "Auth0 Lock" part in every bullet?

File: articles/analytics/integrations/google-analytics/index.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+<script>
+window.auth0AnalyticsOptions = {
+  // Options
+}
+</script>
+<script src="https://cdn.auth0.com/js/lock/10.x/lock.min.js"></script>
+<script src="https://cdn.auth0.com/js/analytics/X.Y.Z/analytics.min.js"></script>
+```
+
+> Notice: The script version above uses a placeholder version `X.Y.Z`. In order to determine the latest release view the [releases in github](https://github.com/auth0/auth0-analytics.js/releases/). For example, to reference release 1.2.0 use `https://cdn.auth0.com/js/analytics/1.2.0/analytics.min.js`
```
Comment (mpaktiti) [inline articles/analytics/integrations/google-analytics/index.md]: @ntotten Can you change this to the format I used in the facebook doc?

```
::: note
The script version above uses a placeholder version `X.Y.Z`. In order to determine the latest release view the [releases in github](https://github.com/auth0/auth0-analytics.js/releases/). For example, to reference release 1.2.0 use `https://cdn.auth0.com/js/analytics/1.2.0/analytics.min.js`
:::
```

For some reason I can't edit this doc using the online editor, the editor closes as soon as it loads.

**Review by mpaktiti — CHANGES_REQUESTED** [review]
Hey Nate, did some minor updates, this only needs one more update at the Google doc (as you can see in my comment there I can't edit this file with the github editor)

**Review by mpaktiti — APPROVED** [review]

File: articles/analytics/integrations/_usage.md (hunk 1 above)
Comment (ntotten) [reply articles/analytics/integrations/_usage.md:6]: Yeah, since its what actually shows up in the event logs.

**Pull request merged**